remotespark/RemoteSparkMagics.py

```diff
@@ -29,94 +29,29 @@ def __init__(self, shell, data=None, mode="normal"):
 
     @magic_arguments()
     @argument("-s", "--sql", type=bool, default=False, help='Whether to use SQL.')
-    @argument("-m", "--mode", help='The mode to execute the magic in: "normal" or "debug". Default is "normal".')
     @argument("-c", "--client", help="The name of the Livy client to use. "
               "Add a session by using %sparkconfig. "
```

Comment:
Also, doc string in line 2, were we outline which magics are offered, needs updating.